tox.ini

```diff
@@ -495,4 +495,4 @@ source =
 directory = ${TOX_ENV_DIR-.}/htmlcov
 
 [coverage:xml]
-output = ${TOX_ENV_DIR-.}/coverage.xml
+output = ${TOX_ENV_DIR-.}/coverage.xml
```

Comment:
This looks like a no-op change-just a line ending so just remove this tox.ini from the changes.